docs/en/changes/changes.md

```diff
@@ -55,6 +55,8 @@
 * Move created the latest index before retrieval indexes by aliases to avoid the 404 exception. This just prevents some interference from manual operations.
 * Add more Go VM metrics, as new skywalking-go agent provided since its 0.2 release.
 * Add component ID for Lock (ID=5016).
+* Apply MQE on Virtual-MQ layer UI-templates
+* Apply MQE on Infra layer UI-templates
```

Comment:
Does Infra layer mean linux and windows? I think there isn't infra layer officially.

Reply:
Sorry, I just looked at the documentation and code and found that there are windows and linux in Infra.
In fact, I only adapted linux in infra. Before this, I thought there was only linux in Infra, because I only saw Linux in the Infra in Live Demo.
I will re-adapt the code to windows and then commit it.

Reply:
Could you rewrite the changelog to keep consistent?